src/main/resources/assets/nochatreports/lang/en_us.json

```diff
@@ -9,5 +9,7 @@
 	"gui.nochatreports.unsafe_server.header": "Caution: Unsafe Server",
 	"gui.nochatreports.unsafe_server.contents": "This server requires all chat messages to be signed, which means they can be reported to Mojang, which may result in a ban from all multiplayer servers. This likely means that the server owner kept \"enforce-secure-profile\" enabled in the server.properties, which is the default since 1.19.1. You can choose to play anyway, but No Chat Reports will not be able to protect you.",
 	"gui.nochatreports.unsafe_server.check": "Do not show again for this server",
-	"gui.nochatrepords.reload_config_tooltip": "Reload No Chat Reports config. Use carefully."
+	"gui.nochatrepords.reload_config_tooltip": "Reload No Chat Reports config. Use carefully.",
+	"modmenu.modrinth": "Modrinth",
+	"modmenu.curseforge": "CurseForge" 
```

Comment:
These two strings are literally inside Mod Menu, you do not need to redefine them.

Reply:
Would it hurt to keep them there?

Reply:
Yes, it is unnecessary and Mod Menu should be the only mod to define those (you are overwriting Mod Menu's text instead of providing custom text for this mod).